Inserted Relationship to Bing Maps Details

cmpendleton-zz opened the pull request:
Request to add relationship with Bing Maps language and table to intro page. #sign-off @dsk-2015 (FYI)

azurecla commented:
Hi __@NarayanAnnamalai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

deneha commented:
@NarayanAnnamalai Thank you for this contribution.  I'll have the author review.

@telmosampaio Would you please review this change and let me know if I should merge it?  Thank you.

Commit:
Inserted Relationship to Bing Maps Details

Request to add relationship with Bing Maps language and table to intro page. #sign-off
